build.gradle

```diff
@@ -657,6 +657,47 @@ project(':iceberg-delta-lake') {
   }
 }
 
+project(':iceberg-bigquery') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation project(':iceberg-gcp')
+
+    implementation platform(libs.google.libraries.bom)
+    compileOnly "com.google.cloud:google-cloud-storage"
+    implementation "com.google.cloud:google-cloud-bigquery"
+    implementation "com.google.cloud:google-cloud-core"
+
+    api project(':iceberg-api')
```

Comment:
nit: you might want to move this and other project dependencies a few lines further above so that project dependencies are grouped together